lexicons/app/bsky/unspecced/getSuggestedUsersSkeleton.json

```diff
@@ -37,6 +37,10 @@
                 "type": "string",
                 "format": "did"
               }
+            },
+            "recId": {
+              "type": "integer",
+              "description": "Snowflake for this recommendation, use when submitting recommendation events."
```

Comment:
Looks like this skeleton method is only used in one place: [getSuggestedUsers](https://github.com/bluesky-social/atproto/blob/d55b4b7b8a2e7974163208f6ab81ee902bbed666/packages/bsky/src/api/app/bsky/unspecced/getSuggestedUsers.ts#L61). So we'll need to add this lex def to that endpoint as well, and pipe the property through to the API response object.